### ISSUE TITLE
Menu

### ISSUE THREAD

**Issue by NachoJusticia** [filippofilip95/gas-oil-mixture-mobile#10]
I don't find the navigation to Options screen much intuitive for the users. I think this kind of buttons are used in Android applications for displaying some additional information, contact, help, etc. Then, I think the menu could be changed/improved.

I think even a simple text saying **Options** is better than the three dots' icon.
![captura de pantalla 2017-11-22 a las 1 27 17](https://user-images.githubusercontent.com/24293766/33103767-97ca67c2-cf24-11e7-939a-e7ecaf14fb27.png)

Anyhow I would develop a new menu using StackNavigator with two tabs initially, Home and Options.  In addition, I think the rate-app or contact messages could be in another screen. 

If you have any ideas regarding this or new features in your mind I would be happy to hear and help you, so you can take me into account for this 👍

**Comment by filippofilip95** [comment]
Hi, @NachoJusticia .
I think that icon in header is good. 
Icon is prettier and clearer than some text. 
Texts on left and right side of header are native used on iOS.

Maybe we could try change three dots icon to some settings icon like this.![settings 1](https://user-images.githubusercontent.com/10739349/33187729-2a8ec8ee-d094-11e7-9c87-e55efb6ac273.png)

**Comment by filippofilip95** [comment]
You are right, that rate-app button and contact messages could be in another screen.
But i think that tabs with home and options, would do this app less straightforward to use than its now.

**Comment by filippofilip95** [comment]
I would like to develop an onboarding for users to use the app.
I mean a real onboarding of app, not a  few screens of app displayed in first start with some text and buttons skip, next.

I mean app onboarding like this.
![inbox-onboarding-experience](https://user-images.githubusercontent.com/10739349/33188240-9f27befa-d098-11e7-9a00-f84ae24572ff.png)
